<ui version="4.0" >
 <class>MediaflyPlayAudio</class>
 <widget class="QWidget" name="MediaflyPlayAudio" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="NScrollLabel" name="m_showTitleLabel" >
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="m_iconLabel" >
         <property name="minimumSize" >
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="m_showDetailsLabel" >
         <property name="font" >
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="NScrollLabel" name="m_episodeNameLabel" >
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NScrollLabel" name="m_numberOfEpisodesLabel" >
       <property name="font" >
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NScrollLabel</class>
   <extends>QLabel</extends>
   <header>nscrolllabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
